recipes/validators/meta.yaml

```diff
@@ -36,22 +36,23 @@ requirements:
   build:
     # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
     # Compilers are named 'c', 'cxx' and 'fortran'.
-    - {{ compiler('c') }}
+    # - {{ compiler('c') }}
   host:
     - python
     - pip
   run:
     - python
+    - decorator
+    - six
 
 test:
   # Some packages might need a `test/commands` key to check CLI.
   # List all the packages/modules that `run_test.py` imports.
   imports:
-    - simplejson
-    - simplejson.tests
+    - validators
```

Comment:
I don't know if it's important, but the Bioconda recipe also tests for importing `validators.i18n`:
https://github.com/bioconda/bioconda-recipes/blob/453ba6844a273f113889d5d3f2a4fb17a1724b48/recipes/validators/meta.yaml#L30